package.json

```
@@ -42,7 +42,7 @@
     "event-loop-spinner": "^2.0.0",
     "gunzip-maybe": "^1.4.2",
     "mkdirp": "^1.0.4",
-    "packageurl-js": "^1.1.1",
+    "packageurl-js": "^1 < 1.2.1",
```

why not pin it to the last good version?
```suggestion
    "packageurl-js": "1.2.0",
```